src/main/java/io/jenkins/plugins/checks/github/GitHubChecksPublisher.java

```diff
@@ -57,15 +59,18 @@ public void publish(final ChecksDetails details) {
             GitHubAppCredentials credentials = context.getCredentials();
             GitHub gitHub = Connector.connect(StringUtils.defaultIfBlank(credentials.getApiUri(), GITHUB_URL),
                     credentials);
-            GHCheckRunBuilder builder = createBuilder(gitHub, new GitHubChecksDetails(details));
-            builder.create();
+
+            GitHubChecksDetails gitHubDetails = new GitHubChecksDetails(details);
+            createBuilder(gitHub, gitHubDetails).create();
             if (listener != null) {
-                listener.getLogger().println("GitHub checks have been published.");
+                listener.getLogger().printf("GitHub check (name: %s, status: %s) has been published.",
+                        gitHubDetails.getName(), gitHubDetails.getStatus());
             }
         }
         catch (IllegalStateException | IOException e) {
             String message = "Failed Publishing GitHub checks: ";
-            LOGGER.log(Level.WARN, message, e);
+            LOGGER.log(Level.WARN,
```

Comment:
can you add a test please? 

either, system-lambda (https://github.com/stefanbirkner/system-lambda)

or I think there's a LogRecorder junit rule in the jenkins test harness are good options